cartoframes/data/observatory/catalog/geography.py

```diff
@@ -154,3 +161,6 @@ def subscription_info(self, credentials=None):
 
         return subscription_info.SubscriptionInfo(
             subscription_info.fetch_subscription_info(self.id, GEOGRAPHY_TYPE, _credentials))
+
+    def is_available_in(self, platform=PLATFORM_BQ):
```

Comment:
Move it to CatalogEntity

Reply:
Same as previous one. It is done there, so I have removed the method from Dataset and Geography classes and download and enrichment are using the method from CatalogEntity